### PR TITLE
Update locale.esES.lua

### PR DESCRIPTION
**Pull request by woopydalan**

### CODE REVIEW

File: Locales/locale.esES.lua [hunk 1]
```diff
@@ -67,13 +67,13 @@ L["New version available! Visit github.com/DungFu/NauticusClassic"] = true
 L["The Thundercaller"] = "El Invocador del Trueno"
 L["The Iron Eagle"] = "El Águila de Hierro"
 L["The Purple Princess"] = "La Princesa Púrpura"
-L["The Maiden's Fancy"] = "La fantasía de la doncella"
+L["The Maiden's Fancy"] = "La Fantasía de la Doncella"
 L["The Bravery"] = "El Valentía"
-L["The Lady Mehley"] = "El Lady Mehley"
+L["The Lady Mehley"] = "El lady Mehley"
```
Comment (DungFu) [inline Locales/locale.esES.lua]: shouldn't this be:
"El Lady Mehley"

**Comment by woopydalan** [comment]
According to Wowhead it’s lady without the capitalization

**Comment by woopydalan** [comment]
Here is a reference if you have some doubt
https://es.wowhead.com/npc=24840/marinera-vines

File: Locales/locale.esES.lua (hunk 1 above)
Comment (woopydalan) [reply Locales/locale.esES.lua]: From wowhead, it appears to not be capitalized
https://es.wowhead.com/npc=24840/marinera-vines

Comment (woopydalan) [reply Locales/locale.esES.lua]: @DungFu Any resolution here? Or just put Lady and change the rest. It doesn't make a big difference.

**Pull request merged**